src/index.js

```diff
@@ -40,9 +81,35 @@ class ToDoList {
       this.removeToDo(toDo);
       toDoApp.remove();
     });
+    const checkBox = toDoApp.querySelector('.checkbox');
+    checkBox.addEventListener('change', () => {
+      this.check(toDoApp);
+      if (toDo.completed === true) {
+        this.toDo.completed = false;
+        console.log(toDo);
+      } else {
+        this.toDo.completed = true;
+        console.log(toDo);
+      }
+    });
+
+    const toDoDescription = toDoApp.querySelector('.todo-description');
+    toDoDescription.addEventListener('dblclick', () => {
+      const editInput = document.createElement('input');
+      editInput.classList.add('edit-input');
+      editInput.value = toDo.description;
+      toDoDescription.replaceWith(editInput);
+      editInput.focus();
+
+      editInput.addEventListener('blur', () => {
+        toDo.description = editInput.value;
+        const newToDoElement = this.createToDo(toDo);
+        toDoApp.replaceWith(newToDoElement);
+      });
+    });
```

Comment:
- [ ] As pointed out by the previous reviewer, even though the edit function works fine, it does not persist the change to the local storage. Please kindly ensure all edit changes are saved